views/update/remove-cannot-use.html

```diff
@@ -0,0 +1,49 @@
+{% extends "update-layout.html" %}
+
+{% block pageTitle %}
+  You cannot apply to de-register this overseas entity - {{ serviceName }} - GOV.UK
```

Comment:
Just wondering, does the tab title display correctly here, using `{{ serviceName }}`? It should reference the remove journey, not update.

Reply:
Copied what was done on https://github.com/companieshouse/overseas-entities-web/pull/1212/files, what do we do to control this I cannot see a service name  parameter anywhere.

Reply:
Maybe service name is not the right thing to put  and should be using what is in the "journey" parameter of the controller.

Reply:
I think `SERVICE_NAME` is what you need to reference. It's set by the code in the `update-layout.html` template.

Reply:
Just run this up - tab does have the correct title.